pkgs/misc/ptouch-print/default.nix

```diff
@@ -1,33 +1,49 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchgit
-, autoreconfHook
+, cmake
+, gettext
 , gd
+, git
 , libusb1
+, pkg-config
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "ptouch-print";
-  version = "1.4.3";
+  version = "1.5-unstable-2024-02-11";
 
   src = fetchgit {
-    url = "https://mockmoon-cybernetics.ch/cgi/cgit/linux/ptouch-print.git";
-    rev = "v${version}";
-    sha256 = "0i57asg2hj1nfwy5lcb0vhrpvb9dqfhf81vh4i929h1kiqhlw2hx";
+    url = "https://git.familie-radermacher.ch/linux/ptouch-print.git";
+    rev = "8aaeecd84b619587dc3885dd4fea4b7310c82fd4";
+    hash = "sha256-IIq3SmMfsgwSYbgG1w/wrBnFtb6xdFK2lkK27Qqk6mw=";
   };
 
   nativeBuildInputs = [
-    autoreconfHook
+    cmake
+    git
+    pkg-config
   ];
 
   buildInputs = [
     gd
+    gettext
     libusb1
```

Comment:
cmake complains abotu some libraries it can't find; these complaints can be mitigated by passing along `libjpeg libpng zlib`